### PR TITLE
DICOM SR and DICOM SEG import export [sc-5791]

### PR DESCRIPTION
**Pull request by wzhlifelover**
-  src code modified from summer intern Dyllan's [work](https://github.com/spike-h/mdai/tree/main) 
- New functions allow SR export, SEG export from MD.ai annotations and SR import into MD.ai annotation note

Examples:
### SR export
exporter = mdai.SR_SEG_utils.SR_SEG_Export('SR', annotation_file, metadata_file)
Successfully exported DICOM SR files into SR_Output
Screen Shot 2023-08-17 at 10 06 33 PM

### SEG export
exporter = mdai.SR_SEG_utils.SR_SEG_Export('SEG', annotation_file, metadata_file)
Successfully exported DICOM SEG files into SEG_Output
Screen Shot 2023-08-17 at 10 05 09 PM

### SR import
mdai_client.import_SR(file_path="DICOM_SR_D_2NnevZ_1.2.826.0.1.3680043.8.498.10501549244263206170119288752475799160_annotator_1.dcm", dataset_id="D_2NnevZ", project_id = "L1Np
nQBv", label_id="L_QRzGlQ")
Importing 1 annotations into project L1NpnQBv, dataset D_2NnevZ...                                  
Successfully imported 1 / 1 annotations into project L1NpnQBv, dataset D_2NnevZ. 
Screen Shot 2023-08-17 at 10 07 09 PM

### CODE REVIEW

File: mdai/__init__.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from .client import Client
 from .utils import common_utils
 from .utils import transforms
+from .utils import SR_SEG_utils
```
Comment (transcranial) [inline mdai/__init__.py:13]: Module names should have short, all-lowercase names (see [pep8](https://peps.python.org/pep-0008/#package-and-module-names)).

**Review by transcranial — CHANGES_REQUESTED** [review]
Module should be called `dicom_utils`. Combining SR with SEG is a bit awkward. How about creating 2 separate classes that inherits a common class?

**Comment by wzhlifelover** [comment]
Reconstructed `dicom_utils` and created two child classes inheriting `DicomExport` Class

> Module should be called `dicom_utils`. Combining SR with SEG is a bit awkward. How about creating 2 separate classes that inherits a common class?

File: mdai/client.py [hunk 2]
```diff
@@ -241,6 +243,127 @@ def import_annotations(
...
+            annotations = []
+            for dicom_dict in referenced_dicoms:
+                study_uid = dicom_dict["Study UID"]
+                note = "\n".join(final_content)
+                annot_dict = {"labelId": label_id, "StudyInstanceUID": study_uid, "note": note}
+                annotations.append(annot_dict)
+
+            self.import_annotations(annotations, project_id, dataset_id)
+
+    def iterate_content_seq(self, content, content_seq_list):
```
Comment (transcranial) [inline mdai/client.py:306]: This should be factored out -- it does not belong here logically.

**Review by transcranial — CHANGES_REQUESTED** [review]

**Review by transcranial — APPROVED** [review]

**Pull request merged**